File: php/fizzbuzz/FizzBuzz.php
<?php

class FizzBuzz {

    public function getFizzBuzzValueOf($value) {
        if ($value % 15 === 0) {
            return  "FizzBuzz";
        }
        if ($value % 3 === 0) {
            return "Fizz";
        }
        if ($value % 5 === 0) {
            return  "Buzz";
        }
        return $value;
    }

}

File: java/fizzbuzz/src/fizzbuzz/FizzBuzz.java
package fizzbuzz;

public class FizzBuzz {

    public String getFizzBuzzValueOf(int number){
        if(number % 15 == 0){
            return "FizzBuzz";
        }
        if(number % 3 == 0){
            return "Fizz";
        }
        if(number % 5 == 0){
            return "Buzz";
        }
        return String.valueOf(number);
    }

}